jest.config.ut.js

```diff
@@ -136,10 +147,10 @@ module.exports = {
   // testLocationInResults: false,
 
   // The glob patterns Jest uses to detect test files
-  testMatch: ['**/__tests__/**/*.(integration.test).[jt]s?(x)'],
+  testMatch: ['**/__tests__/**/user_transformation.test.js'],
```

Comment:
A whole config file just for a single test seems redundant. Let's either move this file to a directory dedicated to UT tests or use `npx jest <file_name>` script to run these tests.

Reply:
updated as per suggestion